css/navigation.css

```diff
@@ -1 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You already have a CSS reset in your `global.css` - you should remove this one and keep the global version only.